### PR TITLE
feat: Initial commit of Project Structure, Components & Storybook

### PR DESCRIPTION
**Pull request by AbhinavDegala**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import "./styles/reset.css";
+@import "./styles/variables.css";
+
+/* body {
```
Comment (ShashankSuresh) [inline src/index.css:4]: Remove this

File: src/components/Span.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+import { compose, space, position, layout, typography } from "styled-system";
+
+const StyledSpan = styled.span`
+  ${compose(space, position, layout, typography)}
```
Comment (ShashankSuresh) [inline src/components/Span.js:5]: Please add font weight here

File: src/stories/Icons.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import { UserIcon, WalletIcon } from "../components/IconFonts";
+import Logo from "../assets/images/Logo.png";
+import FullLogo from "../assets/images/FullLogo.png";
+import { CardHorizontal } from "../components/Card";
+
+export default {
+  title: "Component/Icons",
+};
+
+export const AppLogo = () => <img alt="logo" src={Logo} height="40"></img>;
```
Comment (ShashankSuresh) [inline src/stories/Icons.stories.jsx:11]: Create a component called Logo.js and pass src as its prop

**Review by ShashankSuresh — CHANGES_REQUESTED** [review]

**Comment by AbhinavDegala** [comment]
Made the specified changes

**Pull request closed without merging**